CRM/Member/Form/Membership.php

```diff
@@ -1600,6 +1596,10 @@ protected function emailMembershipReceipt($formValues, $membership) {
       $this->assign('is_pay_later', 0);
       $this->assign('isPrimary', 1);
     }
+    //insert financial type name in receipt.
+    $formValues['contributionType_name'] = CRM_Core_DAO::getFieldValue('CRM_Financial_DAO_FinancialType',
```

Comment:
just a general urgh here for the variable

Reply:
@seamuslee001 just hold your nose & look away